src/System.Security.Cryptography.Algorithms/src/Internal/Cryptography/RandomNumberGeneratorImplementation.OSX.cs

```diff
@@ -2,17 +2,18 @@
 // The .NET Foundation licenses this file to you under the MIT license.
 // See the LICENSE file in the project root for more information.
 
+using System.Diagnostics;
+
 namespace System.Security.Cryptography
 {
     partial class RandomNumberGeneratorImplementation
     {
-        public sealed override void GetBytes(byte[] data)
+        unsafe private void GetBytes(byte* pbBuffer, int count)
```

Comment:
Style: Visibility modifier always comes first, so this should be `private unsafe`